runtime/src/serde_snapshot.rs

```diff
@@ -387,3 +494,117 @@ where
     accounts_db.generate_index(limit_load_slot_count_from_snapshot);
     Ok(accounts_db)
 }
+
+fn reconstruct_accountsdb_from_fields_incremental<E>(
+    fss_accounts_db_fields: AccountsDbFields<E>,
```

Comment:
nit: out of curiosity, what's the second `s`? full snapshot _ / incremental snapshot _?

Reply:
Hah, yeah, so this was for two reasons. One, is sometimes I wanted to talk about snapshots generally with an abbreviation, and a single "S" didn't work for that, so I was doing "SS", i.e. **s**nap**s**hot. Two, I found myself writing variables/comments/debug prints for things like "incremental snapshot slot" or "incremental snapshot hash", etc. With "IS", this ended up as `is_slot`, so it could appear to be a boolean.

I'm also fine fully spelling this vars out as well.

Reply:
> I'm also fine fully spelling this vars out as well.

Yes please fully spell it everything, for the benefit of all future code readers that'll assume ISS means a piece of space junk orbiting the earth, etc.

Reply:
> Yes please fully spell it everything [...]

Will do!

Reply:
Done!